pixl_dcmd/tests/test_database.py

```diff
@@ -86,6 +91,18 @@ def test_get_uniq_pseudo_study_uid_and_update_db(rows_for_database_testing, db_s
     assert pseudo_study_uid == "0.0.0.0.0.0"
 
 
+def test_get_pseudo_patient_id_and_update_db(rows_for_database_testing, db_session):
```

Comment:
we should also have a test explicitly checking for when the database doesn't have a pseudo patient id set, and that the DICOM data's patient ID has been saved to the pseudo patient id column in the database

Reply:
1. Just added [test_with_no_participant_id.csv](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/e11020b89c65e65bc1b3c1a2079e7f2f33d3509d)
[e11020b](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/e11020b89c65e65bc1b3c1a2079e7f2f33d3509d), resulting in this error when using  read_patient_info: `FAILED cli/tests/test_messages_from_files.py::test_non_pseudo_patient_id - ValueError: Export specification files are expected to have at least ['procedure_id', 'participant_id'] as column names.` Do we need another test?

2. For DICOM data's patient ID, is that another test?

Reply:
the column `participant_id` should always exist, just that there should be no value for that column. 

Yep you'll want another test for 2. to confirm that the column in the database has been updated, unless I missed it